pkg/controller/plan/kubevirt.go

```diff
@@ -633,15 +644,15 @@ func (r *KubeVirt) createPodToBindPVCs(vm *plan.VMStatus, pvcNames []string) err
 	pod := &core.Pod{
 		ObjectMeta: meta.ObjectMeta{
 			Namespace:    r.Plan.Spec.TargetNamespace,
-			Labels:       r.vmLabels(vm.Ref),
-			GenerateName: r.getGeneratedName(vm) + "pvcinit",
+			Labels:       r.consumerLabels(vm.Ref),
+			GenerateName: r.getGeneratedName(vm) + "pvcinit-",
 		},
 		Spec: core.PodSpec{
 			RestartPolicy: core.RestartPolicyNever,
 			Containers: []core.Container{
 				{
 					Name:    name,
-					Image:   "ubi9-minimal",
+					Image:   "registry.access.redhat.com/ubi9-minimal:latest",
```

Comment:
@liranr23 thinking about this again, the images are cached on the nodes so maybe it's actually better to use the image of the conversion pod here?

Reply:
ok, that's better - filed #219 for using the right image, not that important though